### ISSUE TITLE
Can't set a value with quotes in it

### ISSUE THREAD

**Issue by bencates** [bitwalker/conform#51]
There is, as far as I can tell, no way to encode the following config option:

```elixir
config :my_app, prop_with_quotes: ~s(foo "bar" baz)
```

**Comment by bitwalker** [comment]
You should be able to do `my_app.prop_with_quotes = foo, "bar", baz`, or `my_app.prop_with_quotes = ["foo", "bar", "baz"]`. I unfortunately can't test right now as I only have a moment, but could you give those a try and report back?

**Comment by bencates** [comment]
Okay, after further testing, it's a slightly more pathological case.  The property I'm trying to set is an API key containing special characters, including both `"` and `#`.

Test cases I tried yesterday:
```
my_app.prop_with_quotes = a"b#c    => "a\"b"
my_app.prop_with_quotes = "a"b#c"  => "ab"
my_app.prop_with_quotes = "a""b#c" => "a\"\"b#c"
my_app.prop_with_quotes = "a\"b#c" => "a\\\"b#c"
```

**Comment by bitwalker** [comment]
This has been fixed and pushed as part of 1.0.0-rc1 :)
